src/jobs/microk8s_install.yml

```diff
@@ -21,32 +41,18 @@ parameters:
     description: "Wait for codacy to be ready (using 'wait_commands')"
     type: boolean
     default: true
-  install_commands:
-    description: "Commands used to install codacy to microk8s"
+  wait_ready_timeout:
```

Comment:
Yes, this is the hardest part, needs a large machine (so we can't use it with github projects, because of our circleci plan for github), and we're already testing functionality elsewhere. Here we're just testing compatibility with kubernetes and helm versions. I'll delete it.